components/02-elements/mega-menu/mega-menu.hbs

```diff
@@ -0,0 +1,55 @@
+<div class="mega-menu">
+    <ul class="mega-menu__wrapper container">
+        {{#each list}}
+            <li class="mega-menu__item {{ class }}">
+                <a href="#" class="mega-menu__link">
+                    {{text}}
+                </a>
+
+                {{#if nestedList}}
+                    <ul class="
+                            mega-menu__dropdown
+                            mega-menu__dropdown--level1
+                        "
+                    >
+                        {{#each nestedList}}
+                            <li class="
+                                    mega-menu__dropdown-item
+                                    mega-menu__dropdown-item--level1
+                                    {{#if modifier}}
+                                        {{modifier}}
+                                    {{/if}}
+                                "
+                            >
+                                <a href="#"
+                                   class="
+                                       mega-menu__link
+                                       mega-menu__link--inner-heading
+                                   "
+                                >
+                                    {{ heading }}
+                                </a>
+
+                                {{> '@list--link'
+                                    listTag="ul"
+                                    elementTag="li"
+                                }}
+                            </li>
+                        {{/each}}
+
+                        <li class="
+                                mega-menu__dropdown-item
+                                mega-menu__dropdown-item--image
+                            "
+                        >
+                            {{> '@image'
+                                src="/images/banner/banner-480_480.png"
+                                additionalClasses="mega-menu__image"
+                            }}
```

Comment:
another element used here... so I'm afraid it should be a module...

Reply:
👍